### PR TITLE
Update the Argo CD custom resource properties table

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
Jira issue: [RHDEVDOCS-6135](https://issues.redhat.com/browse/RHDEVDOCS-6135), [RHDEVDOCS-6133](https://issues.redhat.com/browse/RHDEVDOCS-6133)

Aligned team: DevTools 
Cherry-pick to versions: `gitops-docs-1.14`

Docs preview:

-  [Argo CD custom resource properties](https://81203--ocpdocs-pr.netlify.app/openshift-gitops/latest/argocd_instance/argo-cd-cr-component-properties#argo-cd-properties_argo-cd-cr-component-properties)

SME review: Completed by @varshab1210 
QE review: Completed by @varshab1210
Internal Peer review: @Dhruv-Soni11 

Peer review: 

Additional information: N/A

### CODE REVIEW

File: modules/gitops-argo-cd-properties.adoc [hunk 1]
```diff
@@ -40,7 +40,8 @@ a|* `<processors.operation>` - The number of operation processors.
   * `<sharding.maxShards>` - The maximum number of Argo CD Application Controller replicas.
   * `<sharding.clusterPerShard>` - The number of clusters that need to be managed by each shard. When the replica count reaches the `maxShards`, the shards manage more than one cluster.
   * `<env>` - Environment to set for the application controller workloads.
-  * `sourceNamespaces` - The list of non-control plane namespaces for creating and managing Argo CD `Application` resources in target namespaces.
+  * `<sourceNamespaces>` - The list of non-control plane namespaces for creating and managing Argo CD `Application` resources in target namespaces.
```
Comment (Srivaralakshmi) [inline modules/gitops-argo-cd-properties.adoc:43]: @varshab1210 and @Dhruv-Soni11: FYI, L25, L26, and L43 - 1st column modified to include angular brackets. Earlier the angular brackets were missing.

**Review by dfitzmau — COMMENTED** [review]
/remove-label peer-review-in-progress

/label peer-review-done

**Comment by Dhruv-Soni11** [comment]
The changes you have made to this table look good to me @Srivaralakshmi.

**Pull request merged**